README.md

```diff
@@ -16,60 +17,52 @@ For contract deployment:
 ## Setup
 
 ```bash
-# init and start podman machine
-podman machine init
-podman machine start
-
-# create template yaml
-# NOTE - replace the executor_local_account with your own account address
-podman run --rm \
-  -e pod_name=astria_stack \
-  -e celestia_home_volume=celestia-home-vol \
-  -e metro_home_volume=metro-home-vol \
-  -e executor_home_volume=executor-home-vol \
-  -e relayer_home_volume=relayer-home-vol \
-  -e conductor_home_volume=conductor-home-vol \
-  -e executor_local_account=0xb0E31D878F49Ec0403A25944d6B1aE1bf05D17E1 \
-  -e celestia_app_host_port=26657 \
-  -e bridge_host_port=26659 \
-  -e sequencer_host_port=1318 \
-  -e sequencer_host_grpc_port=9100 \
-  -e executor_host_http_port=8545 \
-  -e executor_host_grpc_port=50051 \
-  -e scripts_host_volume="$PWD"/container-scripts \
-  -v "$PWD"/templates:/data/templates \
-  dcagatay/j2cli:latest \
-  -o /data/templates/astria_stack.yaml \
-  /data/templates/astria_stack.yaml.jinja2
-
-# run pod
-podman play kube --log-level=debug templates/astria_stack.yaml
-
-# deploy test contract
-git clone https://github.com/joroshiba/test-contracts
-cd test-contracts
-export PRIV_KEY=123...
-RUST_LOG=debug forge create --private-key $PRIV_KEY src/Storage.sol:Storage
-
+# create control cluster
+just create-cluster
+
+# ingress
+just deploy-ingress-controller
+# wait for ingress.
+# NOTE: this may fail quickly with "error: no matching resources found".
+#  please retry the command
+just wait-for-ingress-controller
+
+# deploy
+just deploy-astria
 ```
 
+### Connecting Metamask
+* add custom network
+  * network name: `astria-local`
+  * rpc url: `http://executor.astria.localdev.me`
+  * chain id: `1337`
+  * currency symbol: `ETH`
+
 ### Helpful commands
 
 ```bash
-# list running containers
-podman ps
+# logs
+just logs-nginx-controller
+just logs-astria
+just logs-container geth-executor
+
+# list nodes
+kubectl get -n astria-dev-cluster nodes
+
+# list pods
+kubectl get --all-namespaces pods
+kubectl get -n astria-dev-cluster pods
 
-# stop running stack
-podman pod stop astria_stack
+# delete cluster and resources
+just cleanup-astria
```

Comment:
I got this error running this:
```
kind delete cluster --name astria-dev-cluster
Deleting cluster "astria-dev-cluster" ...
Deleted nodes: ["astria-dev-cluster-control-plane"]
kubectl delete -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
E0602 04:35:04.167647 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.168420 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.170390 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.172687 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.173319 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.175502 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.176079 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.178098 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.179993 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.181660 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.183335 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.185326 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.187305 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.189235 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.192097 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.193306 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.194091 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.195668 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
E0602 04:35:04.197378 1977767 memcache.go:265] couldn't get current server API group list: Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
unable to recognize "https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml": Get "http://localhost:8080/api?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
error: Recipe `delete-ingress` failed on line 22 with exit code 1
```

is this normal? does this mean the cluster was still deleted?

Reply:
Yeah, the `cleanup-astria` command does not make sense - after the cluster is deleted none of the other commands are necessary as there is no cluster anymore.

Reply:
I added a better and simpler `cleanup-astria` command! 👍